styles/Canonical/001-English-words-spelling-suggestions.yml

```diff
@@ -30,7 +30,6 @@ swap:
   'defense': defence
   'dialed': dialled
   'dialer': dialler
-  'dialog': dialogue
```

Comment:
annoying though it is 'dialog' is used in lots of GUI tools and I think it's just likely to cause pain and confusion if we try and enforce

Reply:
I agree. "Dialog" is very common.